code/game/objects/items/bodybag.dm

```diff
@@ -86,9 +85,8 @@
 
 /obj/item/bodybag/containment
 	name = "radiation containment body bag"
-	desc = "A folded heavy body bag designed for the storage and transportation of cadavers with heavy radiation."
-	icon = 'icons/obj/bodybag.dmi'
+	desc = "A folded heavy body bag designed for the storage and transportation of heavily irradiated cadavers."
 	icon_state = "radbodybag_folded"
 	unfoldedbag_path = /obj/structure/closet/body_bag/containment
 	w_class = WEIGHT_CLASS_NORMAL
-	item_flags = NO_MAT_REDEMPTION | RAD_NO_CONTAMINATE
+	rad_flags = NO_MAT_REDEMPTION | RAD_NO_CONTAMINATE
```

Comment:
```
	item_flags = NO_MAT_REDEMPTION
	rad_flags = RAD_PROTECT_CONTENTS
```

Reply:
item flag isnt needed as its in the sub-type